cli.js

```diff
@@ -67,6 +67,8 @@ lighthouse({
   } else {
     prettyPrint(results);
   }
+}).catch(function () {
+  process.exit();
```

Comment:
I've test with the version of master branch on node v5.1.1 and defaultUrl was replaced with invalid url as below. exception stack will be printed by reject handler at https://github.com/GoogleChrome/lighthouse/blob/master/index.js#L64 but process still alive
```
const defaultUrl = 'https://m';
```